sdk/storage/azure-storage-file-share/src/test/java/com/azure/storage/file/share/APISpec.groovy

```diff
@@ -43,7 +43,7 @@ class APISpec extends Specification {
     String connectionString
 
     // If debugging is enabled, recordings cannot run as there can only be one proxy at a time.
-    static boolean enableDebugging = false
+    static boolean enableDebugging = true
```

Comment:
revert

Reply:
Should actually probably be removed in favor of the environment variable a la blobs.